executor/cook/executor.py

```diff
@@ -224,9 +277,14 @@ def manage_task(driver, task, stop_signal, completed_signal, config):
     launched_process = None
     task_id = get_task_id(task)
     cio.print_and_log('Starting task {}'.format(task_id))
+    status_updater = StatusUpdater(driver, task_id)
+
+    def inner_os_error_handler(os_error):
+        os_error_handler(os_error, stop_signal, status_updater)
```

Comment:
Nitpick: might be able to accomplish this with `functools.partial`

Reply:
Done. Needed to change the order of the arguments.